### PR TITLE
Patches for issues raised by @LLyaudet in #5 - #22

### PR DESCRIPTION
**Pull request by ingydotnet**

### CODE REVIEW

**Comment by ingydotnet** [comment]
Fixes #5, #6, #7, #8, #9, #10, #11, #12, #13, #14, #15, #19, #20, #21, #24

File: spec/spec.md [hunk 1]
```diff
@@ -6910,17 +6926,17 @@ specification.
 This allows it to evolve to better support YAML [applications].
 Hence, developers are encouraged to submit new "universal" types to the
 repository.
-The yaml-core mailing list at
-<https://lists.sourceforge.net/lists/listinfo/yaml-core>
-is the preferred method for such submissions, as well as raising any questions
-regarding this draft.
+The yaml-core mailing list is the preferred method for such submissions, as
+well as raising any questions regarding this draft.
```
Comment (eemeli) [inline spec/spec.md:6930]: The content of this instruction should be updated, but probably in a separate PR?

**Review by eemeli — APPROVED** [review]

File: spec/spec.md (hunk 1 above)
Comment (ingydotnet) [reply spec/spec.md:6930]: Correct

File: spec/spec.md [hunk 2]
```diff
@@ -1497,21 +1502,23 @@ prefixes.
...
 To ensure readability, YAML [streams] use only the _printable_ subset of the
 Unicode character set.
-The allowed character range explicitly excludes the C0 control block
+The allowed character range explicitly excludes the [C0 control block^]
 **`#x0-#x1F`** (except for TAB **`#x9`**, LF **`#xA`**, and CR **`#xD`** which
 are allowed), DEL **`#x7F`**, the C1 control block **`#x80-#x9F`** (except for
-NEL **`#x85`** which is allowed), the surrogate block **`#xD800-#xDFFF`**,
-**`#xFFFE`**, and **`#xFFFF`**.
+NEL **`#x85`** which is allowed), the [surrogate block^] **`#xD800-#xDFFF`**,
+**`#xFFFE`** and **`#xFFFF`**.
```
Comment (Thom1729) [inline spec/spec.md:1509]: Extreme nitpick, but does the spec has an official position on the Oxford comma?

**Pull request merged**